.bazelrc

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This and other files in this PR need copyright headers. See https://github.com/openxla/stablehlo/blob/main/CMakeLists.txt#L2 for an example.

Reply:
Done.